### PR TITLE
CanaryStrategy forgot to proceed itself

### PR DESCRIPTION
**Pull request by mbdcos**
bug in CanaryStrategy.java

### CODE REVIEW

**Comment by mbdcos** [comment]
retest this please

**Comment by mbdcos** [comment]
retest this please

**Comment by mbdcos** [comment]
retest this please

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/strategy/CanaryStrategy.java [hunk 1]
```diff
@@ -42,8 +42,10 @@
 
     @Override
     public void proceed() {
+        super.proceed();
+        
         if (!initialized.get())  {
-            logger.warn("Proceed has no effect before strategy initialization.");
+            logger.warn("Proceed has no effect to children before strategy initialization.");
```
Comment (mrbrowning) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/strategy/CanaryStrategy.java]: I think the old wording was more intuitive -- I could see this causing users to ask "children of what?"

**Comment by mbdcos** [comment]
retest this please

**Pull request merged**